vicon2_driver_node:
  ros__parameters:
    test_param: "abcdddd"
